services/dcs.yaml

```diff
@@ -1,7 +1,7 @@
-name: dcs
-image: download-controller-service
-description: Interrogation Room Service
+shortname: dcs
+name: download-controller-service
 summary: ""
+version: main
```

Comment:
I think we should specify releases here and not branch names.

Reply:
OK, we can discuss this when it comes to using the descriptions (need a name... service manifests?) and versioning them. Technically I'd expect the tools around them to be able to work with any git ref, including tags of course.